### PR TITLE
Group constraints and weights of cost classes scenario settings dialogs 

### PR DESCRIPTION
**Pull request by hannamocha**

### CODE REVIEW

File: calliope_app/client/component_views/configuration.py [hunk 1]
```diff
@@ -14,6 +14,34 @@
...
+
+    Example:
+    GET: /component/group_constraint_options/
+    """
+
+    model_uuid = request.GET['model_uuid']
+    model = Model.by_uuid(model_uuid)
+    model.handle_view_access(request.user)
+
+    print (str(model) + "")
```
Comment (jgu2) [inline calliope_app/client/component_views/configuration.py:35]: Remove `print` statement.

File: calliope_app/client/templates/scenario_settings.html [hunk 2]
```diff
@@ -47,6 +47,18 @@
 							{% endif %}
 							<input {% if not can_edit %}disabled{% endif %} style="display: none" class="check_delete" type="checkbox" name="delete[{{ param.id }}]">
 						</td>
+
+                        {% if param.run_parameter.id == 25 %}
+                            <td> 
+                                <button {% if not can_edit %}disabled{% endif %} type="button" class="scenario-constraints-dialog-btn btn btn-sm" style="margin-top:7px;" name="dialog[value][{{ param.id }}]"><i class="fas fa-edit"></i></button>
+                            </td>
+                        {% elif param.run_parameter.id == 26 %}
```
Comment (jgu2) [inline calliope_app/client/templates/scenario_settings.html:55]: Hard coded? would you mind put some notes around the code.

Comment (hannamocha) [reply calliope_app/client/templates/scenario_settings.html]: Good call, added a comment! Let me know if you'd like me to implement this differently or put the ids in a settings file.

File: calliope_app/client/component_views/configuration.py (hunk 1 above)
Comment (hannamocha) [reply calliope_app/client/component_views/configuration.py]: Thanks for catching that! Updated.

**Review by jgu2 — APPROVED** [review]

**Pull request merged**